File: RegEx.py
import re

def is_prime(n):
    return not re.match(r'^1?$|^(11+?)\1+$', '1' * n)

print(is_prime(10))  # False
print(is_prime(11))  # True
print(is_prime(12))  # False
print(is_prime(13))  # True

File: RegEx.js
function isPrime(n) {
    return !/^1?$|^(11+?)\1+$/.test('1'.repeat(n));
}

  console.log(isPrime(10));  // false
  console.log(isPrime(11));  // true
  console.log(isPrime(12));  // false
  console.log(isPrime(13));  // true